test-harness/build.gradle

```diff
@@ -18,9 +19,13 @@ dependencies {
 
 	testCompile "com.google.inject.extensions:guice-servlet:${revGuiceServlet}"
 	testCompile "io.swagger:swagger-jersey-jaxrs:${revSwaggerJersey}"
+
+	testCompile "ch.vorburger.mariaDB4j:mariaDB4j:${revMariaDB4j}"
+	testCompile "ch.qos.logback:logback-core:${revLogbackCore}"
+//	testCompile "ch.qos.logback:logback-classic:${revLogbackClassic}"
```

Comment:
Should remove the commented out testCompile include (this was leftover from my PR branch)

Reply:
This is complete